### PR TITLE
chore: replace ESLint and Prettier with Biome

### PR DESCRIPTION
**Pull request by craigzour**
# Summary | Résumé

- Proposal for replacing both ESLint and Prettier with Biome

Note: there are a few small bugs in and there because Biome is still a young tool but it feels very promising.

GT9euwdWgAADusQ

### CODE REVIEW

File: src/routes/forms/submission/new/router.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
-import { Request, Response, Router } from "express";
+import { type Request, type Response, Router } from "express";
 
-const router = Router({
+export const newRouter = Router({
```
Comment (wmoussa-gc) [inline src/routes/forms/submission/new/router.ts:3]: I think the name `newSubmissionRouter` would be more accurate.

**Review by wmoussa-gc — APPROVED** [review]
lots of syntax changes, but good for me

File: package.json [hunk 2]
```diff
@@ -1,46 +1,40 @@
...
-    "nodemon": "^3.1.4",
-    "prettier": "^3.3.3",
-    "supertest": "^7.0.0",
-    "tsx": "^4.16.5",
-    "typescript": "^5.5.4",
-    "typescript-eslint": "^8.0.1",
-    "vitest": "^2.0.5"
-  },
-  "packageManager": "pnpm@9.6.0"
+	"name": "forms-api",
```
[the remark this answers is not in the document]
Comment (craigzour) [reply package.json:2]: I will fix it. While integrating Biome I tested a bunch of things until I found the right starter configuration but then I also ignored this package file so it did not format it with the final configuration which is 
```
"indentStyle": "space",
"indentWidth": 2
```
It does use tabs by default but I figured the team would not be happy with it so I already adjusted it in the configuration.

File: src/routes/forms/submission/new/router.ts (hunk 1 above)
Comment (craigzour) [reply src/routes/forms/submission/new/router.ts:3]: You are right it feels weird. I guess I just wanted to be consistent with the other ones that I renamed. I should try to find another way of naming them then. Will think about it!

**Review by wmoussa-gc — APPROVED** [review]

**Pull request merged**